packages/victory-core/src/victory-animation/victory-animation.js

```diff
@@ -124,15 +123,15 @@ export default class VictoryAnimation extends React.Component {
 
   componentWillUnmount() {
     if (this.loopID) {
-      this.getTimer().unsubscribe(this.loopID);
+      this.timer.unsubscribe(this.loopID);
     } else {
-      this.getTimer().stop();
+      this.timer.stop();
     }
   }
 
   getTimer() {
-    if (this.context.getTimer) {
-      return this.context.getTimer();
+    if (this.context && this.context.globalTimer) {
+      return this.context.globalTimer;
     }
     if (!this.timer) {
```

Comment:
With the new React context, the context should always default to the `globalTimer` defined in `timer-context.js` in case it is not set. Thus, you can assume that this value is set and as such you don't need logic for constructing a timer here.